cmd/plume/fedora.go

```diff
@@ -119,11 +119,11 @@ var (
 )
 
 func AddFedoraSpecFlags(flags *pflag.FlagSet) {
-	flags.StringVarP(&specEnv, "environment", "E", "prod", "instance environment")
-	flags.StringVarP(&specImageType, "image-type", "I", "Cloud-Base", "type of image")
-	flags.StringVarP(&specTimestamp, "timestamp", "T", "", "compose timestamp")
-	flags.StringVarP(&specRespin, "respin", "R", "0", "compose respin")
-	flags.StringVarP(&specComposeID, "compose-id", "O", "", "compose id")
+	flags.StringVar(&specEnv, "environment", "prod", "AMI upload environment")
```

Comment:
Given there are only two modes (the default prod and developer) would it make sense to switch this to a switch and when present swap over to developer partitions?

Reply:
you mean make it a boolean? "true/false" ? I don't mind it like it is but I could see wanting it to be just a boolean.

Reply:
Yeah; ultimately it's more up to you & @jdoss I don't mind leaving it as is.

Reply:
I'll leave it as is then :)